src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+  const rev = Object.values(robot).sort();
+
+  for (let i = 1; i <= rev.length; i++) {
+    if (rev[i] === rev[i - 1]) {
+      return null;
+    }
+  }
+
+  const result = {};
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 3rd item, smth like `inversedRobot` would be more fitting